app-modules/notifications/src/Models/Contracts/Subscribable.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
Planning to add the `subscriptions()` relation requirement here?

Reply:
Ah yeah I can add that in later